pulsedot/usr/share/asteroid-launcher/watchfaces/pulsedot.qml

```diff
@@ -0,0 +1,253 @@
+            var start1 = (minute)*5.9
+            var start2 = (minute)*5.9
+            ctx.reset()
+            ctx.beginPath()
+            ctx.arc(parent.width/2,
+                    parent.height/2,
+                    parent.width * 0.214,
+                    (rot+start1-15) * radian,
+                    (-90+start2-15) * radian, false
+                    );
+            ctx.lineWidth = parent.width * 0.22
+            ctx.strokeStyle = colorctx
+            ctx.stroke()
+        }
+    }
+
+    Rectangle {
+        z:2
+        id: circlePulse
+        antialiasing : true
+        x: root.width/2-width/2
+        y: root.height/2-width/2
+        color: Qt.rgba(0, 0, 0, 1)
+        width: if (wallClock.time.getSeconds() % 2) {parent.width*0.5} else {parent.width*0.65 }
+        height: if (wallClock.time.getSeconds() % 2) {parent.height*0.5} else {parent.height*0.65}
+        radius: width*0.5
+        Behavior on width {
+               NumberAnimation { easing.type: Easing.OutExpo; duration: 1000 }
+        }
+        Behavior on height {
+               NumberAnimation { easing.type: Easing.OutExpo; duration: 1000 }
+        }
+    }
+
+    Rectangle {
+        z:3
+        id: circleDoubleClickCover
+        antialiasing : true
+        property var toggle: 1
+        x: root.width/2-width/2
+        y: root.height/2-width/2
+        color: "black"
+        width: parent.width*0.5
+        height: parent.height*0.5
+        radius: width*0.5
+        MouseArea {
+            anchors.fill: parent
+            onDoubleClicked: {
+               if (circleDoubleClickCover.toggle === 1) {
+                    growH.start()
+                    growW.start()
+                    animateColor2.start()
+                    animateColor3.start()
+                    animateColor4.start()
+                    animateColor5.start()
+                    animateColor6.start()
+                    animateColor7.start()
+                    secondCanvas.colorctx = Qt.rgba(1, 1, 1, 0.6);
+                    secondCanvas.requestPaint();
+                    shrinkH.start()
+                    shrinkW.start()
+                    circleDoubleClickCover.toggle = 0
+               } else {
+                    growH.start()
+                    growW.start()
+                    animateColor2t.start()
+                    animateColor3t.start()
+                    animateColor4t.start()
+                    animateColor5t.start()
+                    animateColor6t.start()
+                    animateColor7t.start()
+                    secondCanvas.colorctx = Qt.rgba(0, 0, 0, 0.6);
+                    secondCanvas.requestPaint();
+                    shrinkH.start()
+                    shrinkW.start()
+                    circleDoubleClickCover.toggle = 1
+                }
+            }
+        }
+        PropertyAnimation {id: growH; target: circleDoubleClickCover; properties: "height"; to: parent.width*0.9; duration: 0}
```

Comment:
Have you considered using a ParallelAnimation ? https://doc.qt.io/qt-5/qml-qtquick-parallelanimation.html basically you can "pack" them and synchronize their starts

Reply:
Not as of yet. 
But thanks for the hint. 
Indeed the animation department looks a bit cleaner too now.